website/markdown-plugin/src/MarkdownFeature.ts

```diff
@@ -7,6 +7,8 @@ import {
 } from "@microsoft/api-documenter";
 import { ApiItem } from "@microsoft/api-extractor-model";
 
+import FunctionJson from "./function.json"; // Same file as /website/function.json. Used to judge path inside markdown-plugin, copy inside because of file path issue after build.
```

Comment:
It's possible that this part will become a debt in the long run. When new methods are added, we need to fix both places, so we'll make sure that the improvement in this part is separately PR.